me.gladwell.eclipse.m2e.android/src/main/java/me/gladwell/eclipse/m2e/android/configuration/AttachDocumentsClasspathLoader.java

```diff
@@ -29,7 +30,7 @@ public AttachDocumentsClasspathLoader(@PrunePlatformProvidedDependencies Classpa
     }
 
     @Override
-    public Iterable<IClasspathEntry> load(IJavaProject project) throws FileNotFoundException {
+    public Iterable<IClasspathEntry> load(IJavaProject project) throws FileNotFoundException, CoreException {
```

Comment:
I not sure what is the better way, to propagate the `CoreException` just like you did with `FileNotFoundException`, or catch locally (and rethrow as `RuntimeException`?)?